en/Installation.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Installation
+---
+
+# Installation
+
+**This help page gets relevant if the [JavaFX](https://builds.jabref.org/javafx/) technology will be released!**
+
+This guide instructs you to install Java in order to run JabRef.
+
+## Necessary Pakages
+
+JavaFX is not included in every Java runtime environment or development kit.
+We highly recommend to use [Oracle Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html), because JavaFX is already included since Java 8.
+
+An alternative to Oracle Java would be OpenJDK with the external library OpenJFX. We only recommend this if you know what you are doing. If you want to use OpenJDK you should follow this [instructions](https://wiki.openjdk.java.net/display/OpenJFX/Building+OpenJFX)
+
+You can download these different versions here:
+
+* [Oracle Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [OpenJDK](http://openjdk.java.net/install/index.html)
+* [OpenJFX](http://packages.ubuntu.com/wily/openjfx-source)
+
+You can check your Java version by typing the following command into your command-line interface:
```

Comment:
Maybe add a sub section "Verify Java installation" and add a comment on how to check if Java is already installed and if so which version.